cmd/remove.go

```diff
@@ -32,20 +31,22 @@ var force bool
 
 // removeCmd represents the remove command
 var removeCmd = &cobra.Command{
-	Use:   "remove",
+	Use:   "remove <stack_name>",
 	Short: "Completely remove a stack",
 	Long: `Completely remove a stack
 
 This command will completely delete a stack, including all of its data
 and configuration. The stack must be stopped to run this command.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if len(args) == 0 {
-			log.Fatal("No stack specified!")
+			return fmt.Errorf("no stack specified")
 		}
 		stackName := args[0]
 
-		if !stacks.CheckExists(stackName) {
-			log.Fatalf("Stack '%s' does not exist!", stackName)
+		if exists, err := stacks.CheckExists(stackName); err != nil {
+			return err
+		} else if !exists {
+			return fmt.Errorf("stack '%s' does not exist", stackName)
 		}
 
 		if !force {
```

Comment:
Should probably move the `force` bool into this function like you do with the flags for the "logs" function. I didn't know how to do it.
(doesn't need to be done now, just noting it for one of us)